src/hub/PolicyHub.Service/Models/PolicyContentRequest.cs

```diff
@@ -31,5 +31,5 @@ public record PolicyContentRequest(
 public record Constraints(
     [property: JsonPropertyName("key")] string Key,
     [property: JsonPropertyName("operator")] OperatorId Operator,
-    [property: JsonPropertyName("value"), JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingNull)] string? Value
+    [property: JsonPropertyName("value"), JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingNull)] IEnumerable<string>? Values
```

Comment:
The change to IEnumerable<string?> isn't mentioned in the issue. Instead the values are posted comma separated. Please change this back to a string? and adjust the logic accordingly.

Reply:
Removed  to IEnumerable<string?> logic changes for comma seperated added.